packages/api/src/command/medical/tcm-encounter/get-tcm-encounters.ts

```diff
@@ -245,7 +253,7 @@ export async function getTcmEncountersCount({
       ...{ coding },
       ...{ status },
       ...{ search: search ? `%${search}%` : "" },
-      ...{ encounterClass },
+      ...(Array.isArray(encounterClass) && encounterClass.length ? { encounterClass } : {}),
```

Comment:
is this necessary? the variable won't be used if the encounter class array is of length 0